### PR TITLE
[WIP]Remove the skeleton/loader for channel switch 

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-19388
#### Checklist

- [ ] Added or updated unit tests (required for all new features)
- [ ] All new/modified APIs include changes to [mattermost-redux](https://github.com/mattermost/mattermost-redux) (please link)
- [ ] Has UI changes
- [ ] Includes text changes and localization file updates

#### Device Information
This PR was tested on:

### CODE REVIEW

**Comment by esethna** [comment]
@sudheerDev what are next steps here in terms of implementation? Is it just a matter of approving the UX or is there more dev work required?

**Comment by sudheerDev** [comment]
@esethna There is one issue i.e when landing on a channel you see a blank screen. Should not be a big issue to fix it if mobile team wants to take it. Other than that i think this is fine.

File: app/components/post_body_additional_content/index.js [hunk 1]
```diff
@@ -45,7 +45,13 @@ function makeMapStateToProps() {
 
     return function mapStateToProps(state, ownProps) {
         const config = getConfig(state);
-        const link = getFirstLink(ownProps.message);
+        const {metadata} = ownProps;
```
Comment (sudheerDev) [inline app/components/post_body_additional_content/index.js]: Note: This is not needed. Had it from another PR.

File: app/screens/channel/channel.ios.js [hunk 2]
```diff
@@ -65,10 +64,6 @@ export default class ChannelIOS extends ChannelBase {
                             valueEvent={CHANNEL_POST_TEXTBOX_VALUE_CHANGE}
                         />
                     </View>
-                    <ChannelLoader
```
Comment (sudheerDev) [inline app/screens/channel/channel.ios.js]: we can remove the entire channel_loader file if we want to go ahead and remove that loader screen

**Comment by sudheerDev** [comment]
Not needed anymore.

**Pull request closed without merging**